Configurations/unix-Makefile.tmpl

```diff
@@ -517,7 +517,7 @@ clean: libclean
 	$(RM) $(PROGRAMS) $(TESTPROGS) $(ENGINES) $(SCRIPTS)
 	$(RM) $(GENERATED_MANDATORY) $(GENERATED)
 	-$(RM) `find . -name '*{- $depext -}' \! -name '.*' \! -type d -print`
-	-$(RM) `find . -name '*{- $objext -}' \! -name '.*' \! -type d -print`
+	find . -name '*{- $objext -}' \! -name '.*' \! -type d -exec $(RM) '{}' \;
```

Comment:
Why does this line need to be changed, but not the line above, which uses the similar construction? Note that this change will cause a lot more processes to be created, slowing down the execution of this target. In addition, the leading `-` has been lost, so that a non-zero exit status from this command will now stop execution of further commands.

Reply:
Yes, I would normally expect this to be done with `xargs`.

Since there are lines in the `Makefile` that use `xargs` already, that would seem the best approach (i.e. it wouldn't be adding a new dependency)

Reply:
I amended the commit to use `xargs` for both lines, tested it and it works. `xargs` on HP-UX 10.20 defaults to `-s 2048`, so no explicit limit parameter to `xargs` is necessary.

Reply:
Yes, I would expect `xargs` on a given system to work within any command-line length restrictions on that system